engine/mpm_solver.py

```diff
@@ -7,7 +7,7 @@
 
 USE_IN_BLENDER = False
 
-ti.require_version(0, 8, 1)
+# ti.require_version(0, 8, 8)
```

Comment:
Didn't find a equivlent function in `0.8.9`

Reply:
That's moved to `ti.init`. Let's simply skip this check as you already have that in `requirements.txt`.